iceoryx_utils/test/moduletests/test_cxx_function_ref.cpp

```diff
@@ -274,3 +286,10 @@ TEST_F(function_refTest, StoreInStdFunctionResultEqual)
     std::function<int()> sut(moep);
     EXPECT_THAT(sut(), Eq(37));
 }
+
+
+TEST_F(function_refTest, CallOverloadedFunctionResultsInCallOfCorrect)
```

Comment:
Could you please also add a test for the void(void) function and check if the value is equal to ```-1```

Reply:
Replaced `void(void)` with `int(void)` to have similar test cases.